### PR TITLE
ansible-galaxy run in a container for stability

### PR DESCRIPTION
**Pull request by literalice**
In the README document, the `run.sh` must need only docker runtime, but actually we need to install ansible-galaxy in a machine directly for executing `run.sh` .

I think `ansible-galaxy` should be run in a container same as `ansible-playbook` command.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -45,10 +43,12 @@ else
     
 fi 
 
+DOCKER_RUN_COMMAND="yum -y install git && ansible-galaxy install -r /tmp/src/requirements.yml --roles-path=/tmp/src/galaxy && $DOCKER_RUN_COMMAND"
```
Comment (literalice) [inline run.sh]: To install roles from git repository, git command have to installed in a container.

* https://github.com/rht-labs/labs-ci-cd/blob/master/galaxy_requirements.yml#L7
* https://github.com/rht-labs/labs-ci-cd/blob/master/requirements.yml#L5

Install git on the base image may be better...
https://github.com/redhat-cop/openshift-applier/blob/master/images/openshift-applier/Dockerfile

**Comment by sherl0cks** [comment]
galaxy is in the container, but there is a regression in the applier container - it no longer ships git. We'll get this fixed upstream. https://github.com/redhat-cop/openshift-applier/issues/18

@oybed :point_up:

**Comment by sherl0cks** [comment]
@literalice can you update the script to consumer `v3.9.0` tag from here instead: https://hub.docker.com/r/redhatcop/openshift-applier/tags/?

That tag should have the bug fixed.

**Comment by oybed** [comment]
@literalice @sherl0cks as the script doesn't specify a specific tag of the container image, this should just work as-is now with the `latest` image. 

However, I'd recommend updating the `galaxy` requirements file to use the `v3.9.0` tag for other purposes.
https://github.com/rht-labs/labs-ci-cd/blob/master/requirements.yml

**Comment by sherl0cks** [comment]
just like we are pinning tags in other places, I'd like to pin it here too

**Comment by literalice** [comment]
@sherl0cks  @oybed Thank you for promptly handling the matter with care!
I agree to specifying the tag for preventing unexpected regression.
I've updated the PR, so would you check it?

**Comment by sherl0cks** [comment]
@literalice I'm running a test now. `run.sh`, due to its nature, doesn't run through our CI. But I am actually testing by recreating the CI infra with your change. Should have feedback shortly.

**Comment by sherl0cks** [comment]
@literalice looks good. Thanks for the change! Do let us know if you have any issues or feature requests.

**Pull request merged**